_data/projects/MindsDB.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: MindsDB
+desc: An Explainable AutoML framework for developers built on top of Pytorch. I
```

Comment:
Typo?

```suggestion
desc: An Explainable AutoML framework for developers built on top of Pytorch.
```

Reply:
@amiexdata  I think MindsDB is more than putting it in a few words. But still, can we try and make it better in the description? Like `An Explainable autoML framework to streamline the use of neural networks built on top of Pytorch`